exasol_transformers_extension/deployment/constants.py

```diff
@@ -16,4 +16,4 @@
         "question_answering_udf.jinja.sql",
 }
 
-
+ORDERED_COLUMNS = "model_name,bucketfs_conn,sub_dir"
```

Comment:
you can also use a list here, because jinja can work with dicts, lists and combinations of it